rpm/qmf-qt5.spec

```diff
@@ -173,6 +173,7 @@ touch .git
     DEFINES+=QMF_NO_MESSAGE_SERVICE_EDITOR \
     DEFINES+=QMF_NO_WIDGETS \
     DEFINES+=USE_ACCOUNTS_QT \
+    DEFINES+=QMF_ACCOUNT_MANAGER_CLASS=SSOAccountManager \
```

Comment:
This part doesn't compile now. Adjusting the name gets better success but would be if build wouldn't need to know class names. There's at least now the USE_ACCOUNTS_QT passed even if the option is halfway automatic and halfway using that.

Reply:
I've removed this finally and use only USE_ACCOUNTS_QT flag. Then, internally, it is setting the QMF_ACCOUNT_MANAGER_CLASS to LibAccountManager. So now it's again fully opt-in.